app/models/manageiq/providers/kubernetes/container_manager/metrics_capture/hawkular_client.rb

```diff
@@ -7,4 +7,8 @@ def initialize(ext_management_system)
 
     hawkular_client
   end
+
+  def system_metrics
+    @client.http_get('/metrics')
```

Comment:
@yaacov I am not sure `system_metrics` is a good name (as above we are using the `_system` tenant by default and it could be misleading).

Is this endpoint returning the definition of the metrics? Can you give an example here (especially WRT the size of the data transferred... I am worried it would be too large).